src/System.ServiceModel.Tests.Common/src/BaseAddress.cs

```diff
@@ -5,6 +5,29 @@
 
 public static class BaseAddress
 {
+#if USE_FIDDLER
+    // Base address for testing nonexistent endpoints
```

Comment:
I agree with @hongdai, @mconnew - it's a stopgap solution to deal with an issue for now. I think the bridge discussion could take a while, so in the meantime, let's make the tests more debuggable for all.